include/pros/motor_group.hpp

```diff
@@ -2357,6 +2357,15 @@ class MotorGroup : public virtual AbstractMotor {
 	 */
 	void append(AbstractMotor&);
 
+
+	/**
+	 * Appends all the motors on the port to this motor group
+	 *
+	 * \param port The port to append to the motor group
+	 *
+	 */
+	void append(std::int8_t port);
+
```

Comment:
Needs some doc updates on rejection conditions (such as if the same absolute value port is found in the motor group)